spec/system/signup/signup_spec.rb

```diff
@@ -0,0 +1,49 @@
+require_relative "../../../lib/google_mail"
+require_relative "../../../lib/eapol_test"
+
+feature "Signup" do
+  include_context "signup"
+
+  it "signs up successfully" do
```

Comment:
Probably better like this?

```suggestion
  it "signs up and authenticates successfully" do
```